mqtt/mqtt-broker/src/broker.rs

```diff
@@ -307,7 +307,7 @@ where
                 let message = Message::Client(client_id, ClientEvent::DropConnection);
                 try_send!(connreq.handle_mut(), message);
             };
-        };
```

Comment:
This change was required for things to properly build. Not sure what is going on.

Reply:
Let's see if mqtt checkin passes.